Online/RainMeadowModManager.cs

```diff
@@ -8,23 +8,46 @@ namespace RainMeadow
     public class RainMeadowModManager
     {
         public static string[] GetActiveMods()
+        { 
+            return ModManager.ActiveMods
+                .Where(mod => Directory.Exists(Path.Combine(mod.path, "modify", "world"))
+                    || highImpactMods.Contains(mod.id)
+                    || cheatMods.Contains(mod.id))
+                .Select(mod => mod.id)
+                .ToArray();
+        }
+        
+        public static readonly string[] highImpactMods = {
+            "rwremix",
+            "moreslugcats",
+            "keepthatawayfromme",  // needs extra syncing to work
+            "no-damage-rng",
+        };
+
+        public static readonly string[] cheatMods = {
+            "devtools",
```

Comment:
what if we moved devtools to high impact

sometimes people just wanna check the debug overlay

Reply:
Sure!